src/components/CompareResults/beta/RevisionSelect.tsx

```diff
@@ -30,9 +30,9 @@ const allRevisionsOption =
 
 const fakeRevisionsOptions = [
   allRevisionsOption,
-  'bb6a5e451dac',
-  '9d5066525489',
-  'a998c42399a8',
+  'bb6a5e451dace3b9c7be42d24c9272738d73e6db',
```

Comment:
now this uses the same mechanism as "normal" queries, we have to use the full `new_rev` value.